content/source/docs/enterprise-beta/saml/login.html.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: enterprise2
+page_title: "SAML Login - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-saml-login"
+---
+
+# Login with SAML
+
+Once SAML is configured users can visit `https://[your-terraform-domain]/session` to login.
```

Comment:
I would like us to standardize on one placeholder decoration! My personal preference is `<ANGLE BRACKET ALLCAPS>`, because it stands out well from basically every kind of surrounding code/config (except XML, which I think we mostly don't use). But I'm not married to it, as long as we're consistent!

Reply:
Only issue with the angle bracket is that they get interpreted as html, so you have to remember to escape them every time.

Reply:
Yup yup. I feel Markdown makes that fairly easy tho, since you can just wrap it in backticks; and since square brackets are used for link syntax in MD, they don't offer much advantage.